packages/backend-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -489,7 +489,7 @@ describe('proxy', () => {
             };
             const proxyOptions = await generateProxyMiddlewareOptions(backend, {}, logger);
             const debugSpy = jest.spyOn(logger, 'debug');
-            debugSpy.mockReset();
+            // debugSpy.mockReset();
```

Comment:
Delete the line instead?

Reply:
Done in 3d67970ea55934680e91cd61e1b191bd65053b33